client/my-sites/plan-features/item.jsx

```diff
@@ -9,31 +9,58 @@ import classNames from 'classnames';
  */
 import Gridicon from 'components/gridicon';
 import InfoPopover from 'components/info-popover';
+import { abtest } from 'lib/abtest';
 
-export default function PlanFeaturesItem( { description, children, className } ) {
-	const renderTipinfo = () => {
-		if ( ! description ) {
-			return null;
-		}
+//export default function PlanFeaturesItem( { description, children, className } ) {
+export default React.createClass( {
+
+	displayName: 'PlanFeaturesItem',
 
+	getInitialState() {
+		return {
+			showPopover: false,
+			mouseOverTimeout: null,
+			mouseOver: false
+		};
+	},
+
+	renderTipinfo( description ) {
 		return (
 			<div className="plan-features__item-tip-info">
 				<InfoPopover
 					position="right"
+					open = { this.state.showPopover }
 				>
 					{ description }
 				</InfoPopover>
 			</div>
 		);
-	};
-
-	const classes = classNames( 'plan-features__item', className );
+	},
+	render() {
+		const { description, children, className } = this.props;
+		const classes = classNames( 'plan-features__item', className );
+		let popoverBindings = {};
+		if ( abtest( 'plansDescriptions' ) === 'auto' ) {
+			popoverBindings = {
```

Comment:
Could we move the abtest check to constructor? If the abtest is set to `auto`, we could assign the `this.mouseOver` and `this.mouseOut` to the particular methods and otherwise set them to noop. I think that could work. :)